backend/chainlit/data/acl.py

```diff
@@ -8,7 +8,7 @@ async def is_thread_author(username: str, thread_id: str):
         raise HTTPException(status_code=400, detail="Data layer not initialized")
 
     thread_author = await data_layer.get_thread_author(thread_id)
-    
+
```

Comment:
Please make sure this is removed from the PR, it is conducive to merge conflicts.